### ISSUE TITLE
Fails when using `wp --path`

### ISSUE THREAD

**Issue by noplanman** [10up/MU-Migration#18]
When using wp-cli with the `--path` flag, the import of content fails (themes, plugins and uploads), because the zip file contains the file structure of the `--path` parameter too.

i.e.
Without `--path`
```
mu-migration-127809963mysite-com.csv
mu-migration-127809963mysite-com.json
mu-migration-127809963mysite-com.sql
wp-content/{plugins,themes,uploads}
```

With `--path="mysite.com"`
```
mu-migration-127809963mysite-com.csv
mu-migration-127809963mysite-com.json
mu-migration-127809963mysite-com.sql
mysite.com/wp-content/{plugins,themes,uploads}
```

So as you can see, the `mysite.com` folder is an extra step when using `--path`, so the proper folders can't be found in [`class-mu-migration-import.php:all()`](https://github.com/10up/MU-Migration/blob/8687160e8168cf329e3da8c1992c5af335211405/includes/commands/class-mu-migration-import.php#L424).

I guess the question now is, should the extra folder depth be there or now.
If not, then the code would need to take `--path` into account.
If yes, then the search for the `wp-content` folder needs to be adjusted.

Either way, I'm happy to PR this.

**Comment by nicholasio** [comment]
The extra folder depth shouldn't be there ideally, but I'm not sure if there's an easy way to remove the extra folder from the zip file without copying the wp-content to the current dir. We can also add the `--path` param to the metadata file so we know the exact path to look for in `import` or just make the detection of `wp-content` smarter in the `import` command. Feel free to craft a PR.

**Comment by noplanman** [comment]
After digging into this, I see that alchemy/zippy causes this issue.
The fact that [they use `getcwd`](https://github.com/alchemy-fr/Zippy/blob/348d258f407d394ea2694b118aff7d33f8a5a33a/src/Adapter/ZipAdapter.php#L66) to define what the tree inside the zip file looks like, it's impossible to tell what folder structure will actually be there.

**Examples**
1. Inside WP root: `wp-content/{plugins,themes,uploads}`
2. One step above WP root: `wp-root/wp-content/{plugins,themes,uploads}` (this is similar for any higher level, it just adds all the subfolders)
3. Some other random folder: `plugins, <theme-name>, uploads` (here `<theme-name>` is the theme folder itself, there is no `themes/<theme-name>` structure here, as the path to be zipped is the theme folder itself.

**BUT**: Since this plugin controls the top level, we can try to figure it out.
Correct me if I'm wrong!

(Numbers same as examples above)
1. One folder called `wp-content`. Easy to find.
2. One folder called whatever the top level is called (in this example `wp-root`. If this is called `plugins` or `uploads` we need a special case. This will never be called `themes`, since the theme folder itself gets added to the zip file right? Unless the theme is called `theme`, which we either ignore or add another special case...)
3. 1-3 folders, depending on what has been chosen to be exported. Handling will be similar to case 2 if only one of the three gets exported. If 2 or 3 are exported, we know what to look for (issue here would be if the theme name is `plugins` or `uploads`, which would just be overwritten).

I hope you understand what I just wrote 😄 

Will try to make a PR to add a smarter detection for the import, stay tuned...

**Comment by noplanman** [comment]
Turns out I was thinking WAY too far... Easy fix #19

**Comment by nicholasio** [comment]
Fixed and merged in #19.
